### PR TITLE
CommunityToolkit/Windows#445

### PR DESCRIPTION
**Pull request by Lightczx**
Fix https://github.com/CommunityToolkit/Windows/issues/445

### CODE REVIEW

File: components/Helpers/src/CommunityToolkit.WinUI.Helpers.csproj [hunk 1]
```diff
@@ -30,4 +30,7 @@
   
   <!-- Sets this up as a toolkit component's source project -->
   <Import Project="$(ToolingDirectory)\ToolkitComponent.SourceProject.props" />
+  <ItemGroup>
+    <None Remove="NativeMethods.json" />
```
Comment (0x5bfa) [inline components/Helpers/src/CommunityToolkit.WinUI.Helpers.csproj]: This is redundant

File: components/Helpers/src/NativeMethods.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "allowMarshaling":  false
```
Comment (0x5bfa) [inline components/Helpers/src/NativeMethods.json]: Add scheme and preserve sig.
```suggestion
  "$schema": "https://aka.ms/CsWin32.schema.json",
  "allowMarshaling": false,
  "comInterop": {
    "preserveSigMethods": [
      "*"
    ]
  }
```

Comment (Lightczx) [reply components/Helpers/src/NativeMethods.json]: We only use
```
RegisterClassEx
DefWindowProc
CreateWindowEx
WM_SETTINGCHANGE
```

Not a single COM interface gets imported.
So I don't think comInterop.preserveSigMethods is needed.

Comment (0x5bfa) [reply components/Helpers/src/NativeMethods.json]: Contributors may use com interfaces for other things. There's no reason to keep it out. Also add the schema.

Comment (Lightczx) [reply components/Helpers/src/NativeMethods.json]: @0x5bfa Done

**Review by 0x5bfa — APPROVED** [review]
Looks good to me 👍
EOF change in components/Helpers/src/MultiTarget.props may have to be reverted.

**Comment by Lightczx** [comment]
@HO-COOH I'm gonna leave this to you.
busy for my own project.

**Pull request merged**